Duplicate error message (esp. when `pretty: true`)

#### What happens and why it is wrong

Duplicate semantic errors get logged to the console, which causes confusion.  For example:

```
./node_modules/rollup/bin/rollup -c ../../rollup.config.js -i ../../js/main.ts -o gen/bundle/main.js --silent
[!] (rpt2 plugin) Error: ../../js/main.ts:8:1 - error TS2304: Cannot find name 'confounded'.

8 confounded;
  ~~~~~~~~~~

../../js/main.ts
Error: ../../js/main.ts:8:1 - error TS2304: Cannot find name 'confounded'.

8 confounded;
  ~~~~~~~~~~

    at error (/Users/kkelly/github/deno/third_party/node_modules/rollup/dist/rollup.js:3365:15)
    at Object.error (/Users/kkelly/github/deno/third_party/node_modules/rollup/dist/rollup.js:20727:13)
    at RollupContext.error (/Users/kkelly/github/deno/third_party/node_modules/rollup-plugin-typescript2/src/rollupcontext.ts:36:18)
    at lodash_2 (/Users/kkelly/github/deno/third_party/node_modules/rollup-plugin-typescript2/src/print-diagnostics.ts:37:10)
    at arrayEach (/Users/kkelly/github/deno/third_party/node_modules/rollup-plugin-typescript2/node_modules/lodash/lodash.js:516:11)
    at forEach (/Users/kkelly/github/deno/third_party/node_modules/rollup-plugin-typescript2/node_modules/lodash/lodash.js:9342:14)
    at printDiagnostics (/Users/kkelly/github/deno/third_party/node_modules/rollup-plugin-typescript2/src/print-diagnostics.ts:9:2)
    at Object.transform (/Users/kkelly/github/deno/third_party/node_modules/rollup-plugin-typescript2/src/index.ts:238:5)
    at /Users/kkelly/github/deno/third_party/node_modules/rollup/dist/rollup.js:20750:41
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

I am not entirely sure how much this is `rollup-plugin-typescript2` and how much of this is just the way rollup behaves, but I couldn't find an answer either way.

#### Versions
- typescript: 3.0.1
- rollup: 0.63.2
- rollup-plugin-typescript2: 0.16.1

I don't get it with just any error, could you create a test repo with reproduction?

Might depend on error type (typescript only, or something that would trip rollup itself) execution mode (clean build / rebuild / watch) and rollup/plugin settings.

This is what I usually get:
(3 `there were errors or warnings` lines for 3 bundles that are generated in that build)

```
rpt2: there were errors or warnings.
rpt2: rolling caches
rpt2: there were errors or warnings.
rpt2: rolling caches
rpt2: there were errors or warnings.
rpt2: rolling caches
(!) Plugin rpt2: src/index.ts:316:18 - error TS2304: Cannot find name 'Plugin5Context'.

316     _onwrite(this: Plugin5Context, _output: OutputOptions): void
                       ~~~~~~~~~~~~~~

src\index.ts
```

--- a/src/index.ts
+++ b/src/index.ts
@@ -261,7 +261,7 @@ const typescript: PluginImpl<RPT2Options> = (options) =>
 			// workaround: err.stack contains err.message and Rollup prints both, causing duplication, so split out the stack itself if it exists (c.f. https://github.com/ezolenko/rollup-plugin-typescript2/issues/103#issuecomment-1172820658)
 			const stackOnly = err.stack?.split(err.message)[1];
 			if (stackOnly)
-				this.error({ message: err.message, stack: stackOnly });
+				this.error({ ...err, message: err.message, stack: stackOnly });
 			else
 				this.error(err);
 		},

--- a/src/index.ts
+++ b/src/index.ts
@@ -253,6 +253,19 @@ const typescript: PluginImpl<RPT2Options> = (options) =>
 			return transformResult;
 		},
 
+		buildEnd(err)
+		{
+			if (!err)
+				return
+
+			// workaround: err.stack contains err.message and Rollup prints both, causing duplication, so split out the stack itself if it exists (c.f. https://github.com/ezolenko/rollup-plugin-typescript2/issues/103#issuecomment-1172820658)
+			const stackOnly = err.stack?.split(err.message)[1];
+			if (stackOnly)
+				this.error({ message: err.message, stack: stackOnly });
+			else
+				this.error(err);
+		},
+
 		generateBundle(bundleOptions)
 		{
 			self._ongenerate();

